instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/coroutines/TestUtils.kt

```diff
@@ -35,6 +37,56 @@ fun sdkTest(
     }
 }
 
+@OptIn(ExperimentalCoroutinesApi::class)
+suspend fun MapboxNavigation.executeActionAndWaitForSessionState(
```

Comment:
maybe nobody needs it outside of this file?
```suggestion
private suspend fun MapboxNavigation.executeActionAndWaitForSessionState(
```